r4/smartconfig/tests/test_service.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+
+@test:Config {groups: ["Service"]}
+function testService() returns error? {
+    http:Client testClient = check new ("http://localhost:9090");
+    json|error response = testClient->get("/fhir/r4/.well-known/smart-configuration");
```

Comment:
Replace with resource access action

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/8693536cd1ee02c420a6325c46a80bc44337ed91